### PR TITLE
German localization

### PR DESCRIPTION
**Pull request by IceDynamix**
We didn't feel like translating the legal and skin stuff (because to be honest who wants to do that), so you'll have to live without them.

Contributors:
@ElectroYan
@IceDynamix

### CODE REVIEW

File: Ranking/Process/de.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+### Unranked
+
+Alle Maps die du zum ersten Mal hochlädst, werden zuerst in den Unranked Status gesetzt. In diesem Status ist es nicht möglich Scores für diese Map hochzuladen, um auf den globalen Ranglisten zu konkurrieren, dennoch ist es möglich sie herunterzuladen.
+
+### Pending
+
+When you submit a mapset for ranking, it will be sent to a queue where our Ranking Supervisors will be able to review the set to make sure it aligns with our Ranking Criteria. 
+
+Wenn du ein Mapset für den Ranked Status einreichst, wird es einer Warteschlange unserer Ranking Supervisors hinzugefügt, von denen sie deine Map auf die Ranking Kriteren überprüfen können.
```
Comment (Swan) [inline Ranking/Process/de.md]: Should be removed

File: Ranking/Process/de.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Ranking Supervisorn ist es nicht erlaubt für ihre eigenen mapsets abzustimmen. Sie müssen durch den gleichen Ablauf wie alle anderen User gehen und auf ihren Platz in der Warteschlange warten.
+
+### 5a. Mapset Abgelehnt
+
+Wenn dein Mapset abgelehnt wurde, wirst du einen Kommentar von einem oder mehreren Supervisors auf der Mapset Webseite bekommen, welche die Ablehnung begründen. Danach ist es dir möglich die Map mit Verbesserungen zu aktualisieren und dannach dann wieder einzureichen. **Behalte im Kopf, dass falls abgelehnt, dein Mapset in einen 7 Tage Cooldown versetzt wird, bis du das Mapset wieder einreichen kannst.
+
+### 5b. Mapset Blacklisted
+
+Serious mapsets will rarely be blacklisted. If your set does reach this state, it means that it was deemed to be in a state where you aren't showing signs of improving the set after repeated denials or if it was never in an approaching rankable state to begin with.
```
Comment (Swan) [inline Ranking/Process/de.md]: Should be removed

File: Gameplay/de.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+| Judgement | Zeitfenster                                  |
+|-------|------------------------------------------------------------|
+|  Marvelous    | ±18ms |
+|  Perfect    | ±43ms |
+|  Great    | ±76ms |
+|  Good    | ±106ms |
+|  Okay    | ±127ms |
+|  Miss   | ±164ms |
+
+When releasing a long note, you are awarded with the `Okay` judgement, which is significantly worse than a miss when it comes to accuracy percentage.
```
Comment (Swan) [inline Gameplay/de.md]: Should be removed

File: Gameplay/de.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+`overall = scoreRating[1] * 95^0 + scoreRating[2] * 95^1 + scoreRating[1] * 95^2 + ...`
+
+#### Punkte
+
+Punkte sind ein anderes Maß deiner Leistung während eines Songs in Quaver. Es ist ein Maß wie gut du darin bist deine Combo, verglichen zu deinen erhaltenen Judgements, zu halten. Es wird in keiner Form für Ranglisten genutzt und dient rein als eine Art von "Erfahrungspunkten". Punktzahl ist auf eine Millionen Punkte limitiert.
+
+#### Judgement Zeitfenster
+
+This section explains the judgement windows that are used within Quaver. It details the time in milliseconds needed to press a key for a note in order to receive it. Currently there is one standard judgement timing window in Quaver, however this may change in the future.
```
Comment (Swan) [inline Gameplay/de.md]: Should be removed

**Comment by IceDynamix** [comment]
Ok should be fine

**Pull request merged**